src/main/flight/mixer.c

```diff
@@ -74,6 +74,7 @@ PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
     .mixerMode = DEFAULT_MIXER,
     .yaw_motors_reversed = false,
     .crashflip_motor_percent = 0,
+    .crashflip_expo = 0,
```

Comment:
Should we pre-set some expo - I think the linear behaviour is not ideal for pretty much all users.

Reply:
> Should we pre-set some expo - I think the linear behaviour is not ideal for pretty much all users.

I was thinking 35% expo, but I have not tested the feel of this. Would people rather have it on the high side or low side?